pyriemann_qiskit/classification.py

```diff
@@ -0,0 +1,360 @@
+"""Module for classification function."""
+import numpy as np
+
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.model_selection import train_test_split
+
+from qiskit import BasicAer, IBMQ
+from qiskit.circuit.library import ZZFeatureMap, TwoLocal
+from qiskit.aqua import QuantumInstance, aqua_globals
+from qiskit.aqua.quantum_instance import logger
+from qiskit.aqua.algorithms import QSVM, SklearnSVM, VQC
+from qiskit.aqua.utils import get_feature_dimension
+from qiskit.providers.ibmq import least_busy
+from qiskit.aqua.components.optimizers import SPSA
+from datetime import datetime
+import logging
+logger.level = logging.INFO
+
+
+class QuanticClassifierBase(BaseEstimator, ClassifierMixin):
+
+    """Quantum classification.
+
+    This class implements a SKLearn wrapper around Qiskit library.
+    It provides a mean to run classification tasks on a local and
+    simulated quantum computer or a remote and real quantum computer.
+    Difference between simulated and real quantum computer will be that:
+
+    * There is no noise on a simulated quantum computer (so results are better)
+    * Real quantum computer are quicker than simulator
+    * Real quantum computer tasks are assigned to a queue
+      before being executed on a back-end
+
+    WARNING: At the moment this implementation only supports binary
+    classification (eg. Target vs Non-Target experiment)
+
+    Parameters
+    ----------
+    target : int
+        Label of the target symbol
+    q_account_token : string (default:None)
```

Comment:
Since `q_account_token` explanation depends on `quantum`, you should describe `quantum` before.